_layouts/default.html

```diff
@@ -67,6 +67,7 @@
                 <li class="dropdown-header">Applicant-Scoped Endpoints</li>
                 <li><a href="custom_field_answer.html">Custom Field Answer</a></li>
                 <li><a href="designation.html">Designation</a></li>
+                <li><a href="social_security_numbers.html">Social Security Numbers</a></li>
```

Comment:
@dheeraj-t-indecomm based on the fact that this request is going to be by Organization off the user identity this should fall in the "User Identity-Scoped Endpoints" category.  Could you fix this.

Reply:
@jcreed Changes done. Please check the updated screenshots in the description of the pull request.